beer-garden/functionary
# #67 - process, store, and display tasking results

Closes #67 

This PR adds a **very basic** implementation of function execution results handling.  It simply does the following:

* Adds two new models: `TaskResult` and `TaskLog`.
* Adds a listener to consume the messages produced by the runner.
* Hands `TASK_RESULT` messages off to a new `record_task_result` celery task.
* That task writes the function results to `TaskResult` and the non-result output to `TaskLog`.

To be clear, this is not an especially scalable solution, but it provides a basic means of showing results for now.

## Test Instructions
The docker-compose file and VSCode config has been updated to include the new listener process.  Just run the "docker-compose up" task and everything that's needed should start.

* Task something via the UI.
* It should redirect you to the task result page upon submission.
* Sit tight for a few seconds and the results should appear.
* You can also use the swagger docs to query for the results or logs via the API.
